src/core/initial_conditions.jl

```diff
@@ -1,3 +1,7 @@
+struct InterStage <: IniCondChronology end
+struct IntraStage <: IniCondChronology end
```

Comment:
Naming:  in these cases I think appending Chronology to the structs (InterStageChronology) increases readability.